src/NUnitEngine/nunit.engine/Services/TestAgency.cs

```diff
@@ -161,10 +161,9 @@ private Guid LaunchAgentProcess(TestPackage package)
 
             string agentExePath = GetTestAgentExePath(useX86Agent);
 
-            if (agentExePath == null)
-                throw new ArgumentException(
-                    string.Format("NUnit components for version {0} of the CLR are not installed",
-                    targetRuntime.ClrVersion.ToString()), "targetRuntime");
+            if (!File.Exists(agentExePath))
+                throw new FileNotFoundException(
+                    $"{Path.GetFileName(agentExePath)} for version {targetRuntime.ClrVersion.ToString()} of the CLR could not be found.", agentExePath);
```

Comment:
The message is a holdover from when we had an optional agent for running .NET 1.1 tests. The agent no longer varies by framework version. We should probably drop the version part of the message and just say we can't find the agent at the path used.